transport/src/main/java/io/zeebe/transport/impl/ClientOutputImpl.java

```diff
@@ -80,22 +80,22 @@ private boolean sendTransportMessage(int remoteStreamId, BufferWriter writer) {
   }
 
   @Override
-  public ActorFuture<ClientResponse> sendRequest(Integer nodeId, BufferWriter writer) {
+  public ActorFuture<ClientResponse> sendRequest(final Integer nodeId, final BufferWriter writer) {
     return sendRequest(nodeId, writer, defaultRequestRetryTimeout);
   }
 
   @Override
   public ActorFuture<ClientResponse> sendRequest(
-      Integer nodeId, BufferWriter writer, Duration timeout) {
-    return sendRequestWithRetry(() -> nodeId, (b) -> false, writer, timeout);
+      final Integer nodeId, final BufferWriter writer, final Duration timeout) {
+    return sendRequestWithRetry(() -> nodeId, this::shouldRetry, writer, timeout);
```

Comment:
Is this change because the transport was previously automatically retrying all `NoRemoteAddressFoundException`?

Reply:
Yes